src/app.css

```
@@ -1,3 +1,8 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@font-face {
+  font-family: 'Inter';
+  src: url('../static/fonts/inter-v12-latin-regular.woff2');
```

```suggestion
  src: url('/fonts/inter-v12-latin-regular.woff2');
```
sveltekit uses "/" to refer to "static" theres no need to use relative paths
refer to these links  https://kit.svelte.dev/docs/project-structure#project-files-static https://stackoverflow.com/a/71848195/9888500

---

It would be nice to have the following variants of the font 200 ,regular, 600,  800